### PR TITLE
fix: adjust global context in flutter

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Description and Example

provide a way to user get global context in flutter:

Example:
`GlobalContext.getInstance().then((value) => value.set("Value Two", "key_two"));`

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.
- [X] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: flutter/beagle/lib/interface/global_context.dart [hunk 1]
```diff
@@ -1,4 +1,19 @@
+import 'package:beagle/service_locator.dart';
+
+/// A Global Context is a class that can assume as value of any type of variable, like a map defines a subset
```
Comment (igorgiumellizup) [inline flutter/beagle/lib/interface/global_context.dart:3]:
```suggestion
/// A Global Context is a class that can assume a value of any variable type. This is similar to a map that defines a subset
```

File: flutter/beagle/lib/interface/global_context.dart [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+/// of key/value or complex JSONs objects that defines object trees.
+///
+/// It works exactly like the Context, however in a global scope, meaning that it will exists while the application is
+/// still running (even on the background), which allows it to be accessed from any application point, being a component
```
Comment (igorgiumellizup) [inline flutter/beagle/lib/interface/global_context.dart:7]:
```suggestion
/// still running (even in the background), which allows it to be accessed from any application point, this point being a component
```

File: flutter/beagle/lib/interface/global_context.dart [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+/// or an action linked to a component or even programmatically.
```
Comment (igorgiumellizup) [inline flutter/beagle/lib/interface/global_context.dart:8]: Would the alteration bellow make sense?
_or an action linked to a component conventionally or programmatically._

File: flutter/beagle/lib/interface/global_context.dart [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
 abstract class GlobalContext {
+
+  /// Gets a current GlobalContext
```
Comment (igorgiumellizup) [inline flutter/beagle/lib/interface/global_context.dart:11]:
```suggestion
  /// Gets the current GlobalContext
```

File: flutter/beagle/lib/interface/global_context.dart [hunk 5]
```diff
@@ -1,4 +1,19 @@
+import 'package:beagle/service_locator.dart';
+
+/// A Global Context is a class that can assume a value of any variable type. This is similar to a map that defines a subset
+/// of key/value or complex JSONs objects that defines object trees.
+///
+/// It works exactly like the Context, however in a global scope, meaning that it will exists while the application is
+/// still running (even in the background), which allows it to be accessed from any application point, this point being a component
+/// or an action linked to a component or even programmatically.
```
Comment (uziasferreirazup) [inline flutter/beagle/lib/interface/global_context.dart:8]:
```suggestion
/// or an action linked to a component conventionally or programmatically.
```

**Review by matheusribeirozup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Pull request merged**